torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -1222,6 +1219,9 @@ def call(self, gm: torch.fx.GraphModule):
         gm = ReplaceMoreTt(gm, self.device, self.use_less_ttnn_op_types).transform()
 
         # Replace patterns manually
-        gm = ReplaceMoreTtManually(gm, self.use_less_ttnn_op_types)
+        while True:
```

Comment:
Maybe set a threshold and throw an exception if exceeds the limit? So the compiler won't silently run infinite loop